Source/WebKitLegacy/mac/WebCoreSupport/WebInspectorClient.mm

```diff
@@ -161,7 +161,7 @@ - (void)destroyInspectorView;
 
     if (enabled) {
         [[m_inspectedWebView window] makeKeyAndOrderFront:nil];
-        [[m_inspectedWebView window] makeFirstResponder:m_inspectedWebView];
+        [[m_inspectedWebView window] makeFirstResponder:m_inspectedWebView.get().get()];
```

Comment:
Using `[m_inspectedWebView window]` with `m_inspectedWebView` as a `WeakObjCPtr<>` causes each method call to autorelease the WebView.

One way to fix this would be to add a convenience method like this, then use that everywhere:

```
    RetainPtr<WebView> inspectedWebView() const { return m_inspectedWebView.get(); }
```

Not necessary to make this change, but I wanted you to be aware of the behavior change.

Reply:
I added a few .get()'s to not add to the autorelease pool.